Tests/HttpPipelineTests/SignedCookieTests.swift

```diff
@@ -41,34 +45,36 @@ aGVsbG8td29ybGQ=\
   }
 
   func testSignedCookie_EncodableValue() {
-    let secret = "cce35c66b1c158d0fdbe93284ab0d2e2003daa0033c4d49753ea8147bdb5a29e30b35d46d5bbad89a6916b9a"
-    let episode = Episode(id: 42, name: "All About Functions")
-    let signedCookieValue = """
+    #if !os(Linux)
```

Comment:
just have to skip the crypto tests on linux :/ 

or hard code the snapshots into the test instead of saving it to a file...